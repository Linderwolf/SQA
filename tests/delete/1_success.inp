admin1
delete
user1