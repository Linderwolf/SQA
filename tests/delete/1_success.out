Welcome to Vapour! Please enter your username:
Username accepted! How may we help you today?
Which user would you like to delete?
Success! Games for sale on this account have been cancelled.
Daily transaction file updated.
